src/pages/patientView/trialMatch/TrialMatchTable.tsx

```diff
@@ -0,0 +1,321 @@
+                                                </span>
+                                            ))}
+                                        </li>
+                                    ))}
+                                </ul>
+                            ) : (
+                                <span>{`${genomicGroupMatch.matches[0].trueHugoSymbol} ${genomicGroupMatch.matches[0].trueProteinChange} `}
+                                    {genomicGroupMatch.matches[0].sampleIds.map((sampleId: string) => (
+                                        <span className={styles.genomicSpan}>
+                                                {this.props.sampleManager!.getComponentForSample(sampleId, 1, '')}
+                                            </span>
+                                    ))}
+                                </span>
+                            )}
+                        </span>
+                    )}
+                    </div>
+                ))}
+                { clinicalGroupMatch.notMatches.length > 0 &&
+                    <DefaultTooltip
+                        placement='bottomLeft'
+                        trigger={['hover', 'focus']}
+                        overlay={this.tooltipGenomicContent(clinicalGroupMatch.notMatches)}
+                        destroyTooltipOnHide={false}
+                        onPopupAlign={placeArrowBottomLeft}
+                    >
+                        {this.mainContent(`No alterations in ${this.getHugoSymbolName(clinicalGroupMatch.notMatches, 3)} defined by the trial`)}
+                    </DefaultTooltip>
+                }
+            </span>
+        );
+    }
+
+    public tooltipGenomicContent(data: Array<IGenomicGroupMatch>) {
+        return (
+            <div className={styles.tooltip}>
+                {data.map((genomicGroupMatch: IGenomicGroupMatch) => (
+                    <div className={styles.genomicSpan}><b>Not </b>{`${genomicGroupMatch.genomicAlteration.replace(/!/g, '')} `}
+                        {genomicGroupMatch.matches[0].sampleIds.map((sampleId: string) => (
+                            <span className={styles.genomicSpan}>
+                                {this.props.sampleManager!.getComponentForSample(sampleId, 1, '')}
+                            </span>
+                        ))}
+                    </div>
+                ))}
+            </div>
+        );
+    }
+    public tooltipClinicalContent(data: Array<string>) {
+        return (
+            <div className={styles.tooltip}>
+                <ul className={styles.alterationUl}>
+                    {data.map((cancerType: string) => (
+                        <li>{cancerType}</li>
+                    ))}
+                </ul>
+            </div>
+        );
+    }
+
+    public mainContent(content: string) {
+        return (
+            <span>
+                <a>{content}</a>
```

Comment:
how come wrapped in <a> tag if not clickeable?

Reply:
It's not clickable but it can be hovered over. I want to use the <a> tag's style to reminder user that this is a link please move cursor on it.

Reply:
but what happens when someone hovers over it?  i mean, i don't see any behavior attached to it